azext_edge/tests/helpers.py

```diff
@@ -7,13 +7,115 @@
 import json
 import os
 import shutil
+from fnmatch import fnmatch
 from knack.log import get_logger
-from typing import Dict
+from typing import Any, Dict, List, Optional, Union
 from azure.cli.core.azclierror import CLIInternalError
+import pytest
+
+from azext_edge.edge.providers.edge_api.base import EdgeResourceApi
 
 logger = get_logger(__name__)
 
 
+def filter_resources(
+    kubectl_items: List[Dict[str, Any]],
+    prefixes: Optional[Union[str, List[str]]] = None,
+    resource_match: Optional[str] = None,
+) -> Dict[str, Any]:
+    def pass_conditions(item: Dict[str, Any]) -> bool:
+        name = item["metadata"]["name"]
+        if resource_match and not fnmatch(name, resource_match):
```

Comment:
should be a simple return

Reply:
ryan commented:
![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/73560279/53503721-67ae-4f50-abc4-075c9a44f932)


so I will think about what would improve readability the most and use that